client/src/Components/App.jsx

```diff
@@ -1,10 +1,26 @@
 import React from 'react';
+import Stars from './Stars.jsx';
+import RatingsContainer from './RatingsContainer.jsx';
+import Reviews from './Reviews.jsx';
 
 const App = () => {
 
+
+  let [users, getUsers] = React.useState([]);
```

Comment:
hooks! very nice

Reply:
that's the plan. just made the components where data from dB will go. then pass em along as props

Reply:
I've been learning hooks. for a couple of days now. hopefully I get it righ